packages/notebook-extension/src/index.ts

```diff
@@ -378,6 +378,42 @@ export const notebookTrustItem: JupyterFrontEndPlugin<void> = {
   }
 };
 
+export const toggleHeader: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/apputils-extension:toggle-header',
```

Comment:
Given that this toggles the header on any MainAreaWidget (e.g., the grid viewer, the console, etc.), this command probably belongs in apputils-extension rather than the notebook extension.

Reply:
Thank you for bearing with me 😅, and for the pointer 🙏, fixed!!!